### DISCUSSION TITLE
Ways to disable cycle dependency error

### DISCUSSION THREAD

**Discussion by Sravanthi1424** [webdriverio/webdriverio#10093] [Community]
Hi, the disable statement /* eslint-disable import/no-cycle */ is not working now. Please suggest how to disable the cycle dependency error. Thank you.

**Comment by christian-bromann** [comment]
I am not sure how this is related to WebdriverIO 🤔 I don't know your code, so it is impossible to provide you support.

**Reply by Sravanthi1424** [reply]
Hi @christian-bromann , first of all thanks much for the reply. Yes this is not related to webdriverio but I am new bie to webdriverio + typescript. I found no source in google to handle the above error. Just trying out my luck here if anyone can give pointers on how to disable the below error.

**Reply by christian-bromann** [reply]
If you don't know what this eslint error means I recommend to disable it
